src/components/AddUsersModal/emailTemplate.js

```diff
@@ -0,0 +1,14 @@
+const emailTemplate = {
+  greeting: `Your Learning Manager has provided you with edX course subscription.
```

Comment:
correct this to read: "...provided you with an edX course...", adding "an" to the phrasing.

Reply:
@moconnell1453 This change is related to AddUserModal and being done in [PR](https://github.com/edx/frontend-app-admin-portal/pull/268). I have made my PR from those changes but looks like some changes are needed there yet.
That's why I am closing this PR and opening a new PR which contains only my changes.